samples/bluetooth/broadcast_audio_source/boards/nrf5340_audio_dk_nrf5340_cpuapp.conf

```diff
@@ -10,3 +10,17 @@ CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
 CONFIG_NEWLIB_LIBC=y
 CONFIG_MAIN_STACK_SIZE=4096
 CONFIG_BT_TINYCRYPT_ECC=y
+
+# USB Audio related configs
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_PRODUCT="Zephyr Broadcast Source"
+CONFIG_USB_DEVICE_INITIALIZE_AT_BOOT=n
+CONFIG_USB_DEVICE_AUDIO=y
+
+# Only stream one channel
+CONFIG_BT_ISO_MAX_CHAN=1
+CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT=1
+CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT=1
```

Comment:
I had limited it to one because of performance issues - but considering to reintroducing 2 (making it stereo) with an update to the PR that also offloads the encoding to a separate thread